### PR TITLE
Command line grid

### PR DESCRIPTION
**Pull request by anatoli-karotki**

### CODE REVIEW

**Comment by apixandru** [comment]
you should specify java version in pom file

File: src/main/java/dr/anatolik/controller/Controller.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    private final Grid grid = new Grid();
+    private final Printer printer = new Printer();
+
+
+    public void startDialog() {
+        System.out.println(printer.printGrid(grid));
+        System.out.println("\nSelect number 0 - 99");
+        int answer = getIntAnswer();
+        grid.updateCell(answer);
+        startDialog();
```
Comment (apixandru) [inline src/main/java/dr/anatolik/controller/Controller.java:21]: better to not use recursion but loop

File: src/main/java/dr/anatolik/controller/Controller.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+public class Controller {
+
+    private final Grid grid = new Grid();
+    private final Printer printer = new Printer();
+
+
+    public void startDialog() {
+        System.out.println(printer.printGrid(grid));
+        System.out.println("\nSelect number 0 - 99");
```
Comment (apixandru) [inline src/main/java/dr/anatolik/controller/Controller.java:18]: should prefer to use print vs println when expecting input.

**Review by apixandru — APPROVED** [review]

File: src/main/java/dr/anatolik/controller/Controller.java (hunk 2 above)
Comment (anatoli-karotki) [reply src/main/java/dr/anatolik/controller/Controller.java:18]: @apixandru  Why it is better?

Comment (apixandru) [reply src/main/java/dr/anatolik/controller/Controller.java:18]: normally any cli application asks for a reply on the same line as the question, not on a new line

examples:
https://support.eapps.com/eapps_images/cloud/con_ssh_img/cnnct_ssh_id.png
https://linuxhint.com/wp-content/uploads/2019/10/9-13.png
https://www.howtogeek.com/wp-content/uploads/2016/06/img_5775d608ceb5e.png
https://www.howtogeek.com/wp-content/uploads/2014/03/06_rm_confirmation.png

**Pull request merged**